berkeley-function-call-leaderboard/model_handler/constant.py

```diff
@@ -161,3 +162,9 @@
     "rest": "gorilla_openfunctions_v1_test_rest.json",
     "sql": "gorilla_openfunctions_v1_test_sql.json",
 }
+
+FORMAT_REMINDER = """Should you decide to return the function call(s),Put it in the format of [func1(params_name=params_value, params_name2=params_value2...), func2(params)]\n
```

Comment:
Thanks for the PR @HectorFake We are converging towards a constant system prompt and user prompt for all models. Can we just rely on that? Or is this FORMAT_REMINDER absolutely necessary? If you think it might be useful for users, then we can keep it, although for the BFCL itself, we plan to use a very consistent prompt to evaluate all models.